en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -334,6 +334,103 @@ When there are null values in custom expressions and are:
 * Fields without attributes, the null value is ignored. If the value in a required field, an error message occurs. If the value is in an optional field, the value is ignored and not sent to the connector. +
 * Fields with attributes, the value is ignored and not sent to the connector.
 
+== Define a Variable
+
+You can define variables in your Composer flow. Defining a variable allows you to complete certain use cases like aggregating calculations and storing values. Variables are defined locally within a flow and can be used anywhere within that flow. 
+
+[[access-the-variable-step]]
+=== Access the Variable Step 
+
+To access the *Variable* step: 
+
+. <<create-a-new-flow,Create a new flow>> or open an existing flow. +
```

Comment:
I think we decided to use the + on a line by itself for consistent line spacing.

Reply:
Done! 

PS - I wonder if there is a find and replace function we can use to fix this... because Composer is full of blank spaces.